### PR TITLE
Training

### PR DESCRIPTION
**Pull request by abrorAbdullaev**

### CODE REVIEW

File: app/Http/Controllers/Slider/IndexController.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public function __construct(SliderManager $sliderManager)
+    {
+        $this->sliderManager = $sliderManager;
+    }
+
+    /**
+     * Main action of the app, returns the view of the slider passing the params of the
+     * slides
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
```
Comment (faik371) [inline app/Http/Controllers/Slider/IndexController.php]: Import class

File: app/Migrations/Slider/2018_02_08_170248_create_sliders_table.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{
+    /**
+     * Creates the sliders table
+     */
+    public function up()
+    {
+        Schema::create('sliders', function(Blueprint $table) {
+            $table->increments('id');
+            $table->date('date');
+
```
Comment (faik371) [inline app/Migrations/Slider/2018_02_08_170248_create_sliders_table.php]: why empty line?

File: app/Models/Slider/Slider.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+     *
+     * @param string(date)|null $date
+     * @param array $attributes
+     */
+    public function __construct(
+        $date = null,
+        array $attributes = []
+    ) {
+        $this->date = $date ? $date : date(SliderManager::DATE_TYPE_FORMAT);
+        parent::__construct($attributes);
```
Comment (faik371) [inline app/Models/Slider/Slider.php]: empty line before return

File: resources/views/includes/footer.blade.php [hunk 4]
```diff
@@ -0,0 +1,2 @@
+{{-- All the content was done in the scope of training and can be updated withouth discussion if needed --}}
+<h3 id="copyright text-right">© Copyright 2018 InforMe team</h3>
```
Comment (faik371) [inline resources/views/includes/footer.blade.php]: hardcoded year

File: resources/views/includes/head.blade.php [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<meta charset="utf-8">
+<meta name="description" content="InforMe Slider">
+<meta name="author" content="InforMe Team">
+<meta name="title" content="InforMe">
+
+{{-- TODO: Implement dynamic titles --}}
+<title>Dummy title</title>
+
+{{-- TODO: Check whether the stylesheets paths are correct --}}
+<link rel="stylesheet" href="../../../public/css/main.css">
```
Comment (faik371) [inline resources/views/includes/head.blade.php]: use {{ asset('absolute path from public folder') }} blade function

Comment (abrorAbdullaev) [reply resources/views/includes/head.blade.php]: This one is only usefull comment :)

Comment (faik371) [reply resources/views/includes/head.blade.php]: blah blah blah. read PSR-2 :)

**Comment by faik371** [comment]
abrorAbdullaev deleted a comment from faik371 7 days ago

**Pull request closed without merging**